src/mgmt/operator/obj/droplet.py

```diff
@@ -48,10 +50,13 @@ def rpc(self):
 
     def get_obj_spec(self):
         self.obj = {
+            "name": self.name,
```

Comment:
We don't need this as the name is in the metadata.

Reply:
fixed